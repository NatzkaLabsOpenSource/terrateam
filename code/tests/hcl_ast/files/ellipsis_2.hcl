foo = {for i, v in ["a", "a", "b"]: v => i...}

