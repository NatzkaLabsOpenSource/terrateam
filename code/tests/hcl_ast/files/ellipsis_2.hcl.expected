[
  [
    "Attribute",
    "foo",
    [
      "For_object",
      {
        "identifiers": [ "i", [ "v" ] ],
        "input": [
          "Tuple",
          [ [ "String", "a" ], [ "String", "a" ], [ "String", "b" ] ]
        ],
        "key_output": [ "Id", "v" ],
        "value_output": [ "Ellipsis", [ "Id", "i" ] ],
        "cond": null
      }
    ]
  ]
]
